webtool/frontend/src/components/heat-storage/heat-storage-display.tsx

```diff
@@ -1,13 +1,20 @@
 import {FunctionComponent} from "react"
-import {Card, DataList} from "@radix-ui/themes"
+import {Flex, Card, DataList} from "@radix-ui/themes"
 import {HeatStorage} from "local4local"
 import {HeatStorageHeading} from "./heat-storage-heading.tsx"
 import {CostDisplay} from "../cost-section.tsx"
+import {CardMenu} from "./../card-menu.tsx"
 
-export const HeatStorageDisplay: FunctionComponent<{ heatStorage: HeatStorage }> = ({heatStorage}) => {
+export const HeatStorageDisplay: FunctionComponent<{
+        heatStorage: HeatStorage,
+        toDelete: () => void,
+    }> = ({heatStorage, toDelete}) => {
     return (
         <Card>
-            <HeatStorageHeading />
+            <Flex gap="3">
```

Comment:
If you use `justify="space-between"` here then you don't need to use `position: absolute` for the CardMenu